.github/workflows/pimcore-demo.yml

```
@@ -20,10 +20,10 @@ jobs:
     strategy:
         matrix:
             include:
-                - { php-version: 8.2, database: "mariadb:10.11", pimcore_version: "", experimental: true }
-                - { php-version: 8.2, database: "mariadb:10.11", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
-                - { php-version: 8.2, database: "mysql:latest", pimcore_version: "", experimental: true }
-                - { php-version: 8.2, database: "mysql:latest", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
+                - { php-version: 8.3, database: "mariadb:10.11", pimcore_version: "", experimental: true }
+                - { php-version: 8.3, database: "mariadb:10.11", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
+                - { php-version: 8.3, database: "mysql:latest", pimcore_version: "", experimental: true }
+                - { php-version: 8.3, database: "mysql:latest", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
```

```suggestion
                - { php-version: 8.3, database: "mariadb:10.11", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.3.99", experimental: true }
                - { php-version: 8.3, database: "mysql:latest", pimcore_version: "", experimental: true }
                - { php-version: 8.3, database: "mysql:latest", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.3.99", experimental: true }
```
the change in https://github.com/pimcore/admin-ui-classic-bundle/pull/600/files forces to not work with eg. `11.99.0`
it would be restored by https://github.com/pimcore/admin-ui-classic-bundle/pull/601/files

---

You could also add a branch alias for pimcore/pimcore like https://github.com/pimcore/admin-ui-classic-bundle/pull/594

Than you can remove the "as 11.0.0" and the test knows the right version.

---

that is a nice idea, and we could clean all the alias usages, which somehow got inconsistent, somewhere is `11.0.99`, some other places is `11.99.99`